Util/Random/random_between.m

```diff
@@ -0,0 +1,53 @@
+function [arr] = random_between(a,b,n,type)
```

Comment:
same as above and other function names in this PR. but why those "random" functions? where are they used?

Reply:
as above for renaming. I used them on some unpushed tests and for automatic generation of some things. Typically, these include checking randomly cell indexes.